cmd/thanos/bucket.go

```diff
@@ -49,18 +47,38 @@ func registerBucket(m map[string]setupFunc, app *kingpin.Application, name strin
 	verify := cmd.Command("verify", "verify all blocks in the bucket against specified issues")
 	verifyRepair := verify.Flag("repair", "attempt to repair blocks for which issues were detected").
 		Short('r').Default("false").Bool()
+	// NOTE(bplotka): Currently we support backup buckets only in the same project.
+	verifyBackupGCSBucket := cmd.Flag("gcs-backup-bucket", "Google Cloud Storage bucket name to backup blocks on repair operations.").
+		PlaceHolder("<bucket>").String()
+	verifyBackupS3Bucket := cmd.Flag("s3-backup-bucket", "S3 bucket name to backup blocks on repair operations.").
+		PlaceHolder("<bucket>").String()
```

Comment:
How long until someone requests to provide separate credentials for these :P?

Reply:
I bet -> on next compactor bug (if any)
Hopefully, at that point we will have shiny https://github.com/improbable-eng/thanos/issues/209

(: